mw-webapp/src/dictionary/WayPageContent.json

```diff
@@ -591,6 +591,12 @@
         "ua": "Натисніть, щоб позначити план як завершений",
         "ka": "დააწკაპუნეთ გეგმის დასრულებულად მონიშვნაზე"
       },
+      "planCheckboxUncompleted": {
+        "ru": "Нажмите, чтобы отметить план как невыполненный",
+        "en": "Click to mark the plan as uncompleted",
+        "ua": "Натисніть, щоб позначити план як незавершений",
+        "ka": "დააწკაპუნეთ გეგმის დაუსრულებლად მონიშვნაზე"
+      },
```

Comment:
Please xcheck do we use anywhere lines 588-593, 600-611?

Reply:
yes , we use this lines